apps/prairielearn/src/schemas/schemas/infoAssessment.json

```
@@ -398,87 +415,127 @@
         }
       }
     },
-    "questionAlternative": {
+    "QuestionAlternativeJsonSchema": {
       "type": "object",
+      "required": ["id"],
       "properties": {
         "comment": {
           "description": "Arbitrary comment for reference purposes.",
-          "anyOf": [{ "type": "string" }, { "type": "array" }, { "type": "object" }]
+          "$ref": "#/definitions/CommentJsonSchema"
         },
         "points": {
-          "$ref": "#/definitions/points"
+          "$ref": "#/definitions/PointsJsonSchema"
         },
         "autoPoints": {
-          "$ref": "#/definitions/points"
+          "$ref": "#/definitions/PointsJsonSchema"
         },
         "maxPoints": {
-          "$ref": "#/definitions/pointsSingle"
+          "description": "A single point value.",
+          "$ref": "#/definitions/PointsSingleJsonSchema"
         },
         "maxAutoPoints": {
-          "$ref": "#/definitions/pointsSingle"
+          "description": "A single point value.",
+          "$ref": "#/definitions/PointsSingleJsonSchema"
         },
         "manualPoints": {
-          "$ref": "#/definitions/pointsSingle"
+          "description": "A single point value.",
+          "$ref": "#/definitions/PointsSingleJsonSchema"
         },
         "id": {
-          "$ref": "#/definitions/questionId"
+          "$ref": "#/definitions/QuestionIdJsonSchema"
         },
         "forceMaxPoints": {
-          "$ref": "#/definitions/forceMaxPoints"
+          "description": "Whether to force this question to be awarded maximum points on a regrade.",
+          "$ref": "#/definitions/ForceMaxPointsJsonSchema"
         },
         "triesPerVariant": {
           "description": "The maximum number of graded submissions allowed for each question instance.",
           "type": "number",
           "default": 1
         },
         "advanceScorePerc": {
-          "$ref": "#/definitions/advanceScorePerc"
+          "description": "Minimum score percentage to unlock access to subsequent questions",
```

Let's change the corresponding Zod schema:

```suggestion
          "description": "Minimum score percentage to unlock access to subsequent questions.",
```

---

Done in d41984e.